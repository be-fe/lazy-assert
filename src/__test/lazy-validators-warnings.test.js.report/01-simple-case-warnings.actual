[]
  0 : {}
    0 : "[WARN]  peek <01 1, undefined> Validation failed with the given validator, the result is :"
  1 : {}
    0 : {}
      expected : undefined
      message : "Target is not equal to the expected value."
      type : "exact-equal"
  2 : {}
    0 : "[WARN]  peek <01 1, undefined> A default validator is : "
  3 : {}
    0 : "\"number\""
  4 : {}
    0 : "[WARN]  peek <02 1, \"object\"> Validation failed with the given validator, the result is :"
  5 : {}
    0 : {}
      message : "Target value is not of the expected type"
      type : "type.object"
  6 : {}
    0 : "[WARN]  peek <02 1, \"object\"> A default validator is : "
  7 : {}
    0 : "\"number\""
  8 : {}
    0 : "[WARN]  peek <03 function, [array, number]> Validation failed with the given validator, the result is :"
  9 : {}
    0 : {}
      message : "Target is not an array."
      type : "array-array.target-not-array"
  10 : {}
    0 : "[WARN]  peek <03 function, [array, number]> A default validator is : "
  11 : {}
    0 : "\"function\""