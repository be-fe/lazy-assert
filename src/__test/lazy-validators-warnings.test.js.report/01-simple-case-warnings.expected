[]
  0 : {}
    0 : "[WARN]  peek <01 1, undefined> Validation failed with the given validator, the result is :"
  1 : {}
    0 : "$VAL : $COND => Target is not equal to the expected value. <Expected value: undefined >"
  2 : {}
    0 : "[WARN]  peek <01 1, undefined> A default validator is : "
  3 : {}
    0 : "\"number\""
  4 : {}
    0 : "[WARN]  peek <02 1, \"object\"> Validation failed with the given validator, the result is :"
  5 : {}
    0 : "$VAL : $COND => Target value is not of the expected type <Expected type: object >"
  6 : {}
    0 : "[WARN]  peek <02 1, \"object\"> A default validator is : "
  7 : {}
    0 : "\"number\""
  8 : {}
    0 : "[WARN]  peek <03 function, [array, number]> Validation failed with the given validator, the result is :"
  9 : {}
    0 : "$VAL : $COND => Target is not an array."
  10 : {}
    0 : "[WARN]  peek <03 function, [array, number]> A default validator is : "
  11 : {}
    0 : "\"function\""